### DISCUSSION TITLE
duck cli: Switch to pass a password for a Cryptomator vault

### DISCUSSION THREAD

**Discussion by br0kenpixel** [iterate-ch/cyberduck#15370] [Q&A]
Is there a switch (or some other way) of passing a Cryptomator vault password to `duck`? The `-p` switch is used for logging in to my WebDAV server only.

**Comment by dkocher** [comment]
Unfortunately there is no such option currently to access vaults [^1] from Cyberduck CLI.

[^1]: https://docs.duck.sh/cli/#cryptomator
